app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1> Lauren's Tasklist:</h1>
+
+<main id="list">
+  <h2> Tasks</h2>
+
+  <ol >
+      <%= render partial: "task_summary", locals: {action: "task_summary"} %>
```

Comment:
Since we're only using the `task_summary` partial in this location we should probably just put the code from that file in this spot directly. Partials help us DRY up our view code, but only if we are using them in multiple places.

Additionally, we're not using the `action` local variable within the `task_summary` partial, so we could also have left off the whole `locals` section of this line.